lib/datasource/repology/index.ts

```diff
@@ -0,0 +1,126 @@
+import { URLSearchParams } from 'url';
+import { logger } from '../../logger';
+import * as globalCache from '../../util/cache/global';
+import got from '../../util/got';
```

Comment:
Use `Http` wrapper instead of `got`

```suggestion
import { Http } from '../../util/http';
```

https://github.com/renovatebot/renovate/blob/f652a8c7d6cd3f7316d91f8f0f7f0246b7bb0a8e/lib/util/http/index.ts#L31

Reply:
Adjusted, the datasource now uses the `Http` wrapper instead. Thanks!